python/paddle/fluid/tests/unittests/test_jit_save_load.py

```diff
@@ -986,5 +1030,105 @@ def test_jit_save_data_parallel_with_to_static(self):
         self.verify_inference_correctness(layer, path)
 
 
+class InputSepcLayer(paddle.nn.Layer):
+    '''
+    A layer with InputSpec to test InputSpec compatibility
+    '''
+
+    @paddle.jit.to_static(input_spec=[
+        InputSpec(
+            shape=[None, 8], dtype='float32', name='x'), InputSpec(
+                shape=[None, 1], dtype='float64', name='y')
+    ])
+    def forward(self, x, y):
+        return x, y
+
+
+class TestInputSpecCompatibility(unittest.TestCase):
+    def _assert_input_spec_layer_return(self, expect_layer, test_layer):
+        input_x = paddle.uniform([8, 8], dtype='float32')
+        input_y = paddle.uniform([8, 1], dtype='float64')
+        expected_result = expect_layer(input_x, input_y)
+        test_result = test_layer(input_x, input_y)
+        self.assertTrue(
+            float((expected_result[0] - test_result[0]).abs().max()) < 1e-5)
+        self.assertTrue(
+            float((expected_result[1] - test_result[1]).abs().max()) < 1e-5)
```

Comment:
A small question: Why not to use `np.array_equal` or `np.allclose` here? They will check both `rtol` and `atol`.

Reply:
This was just copied from tests in same file, I have changed this test case.